WORKSPACE

```diff
@@ -212,6 +216,11 @@ distdir_tar(
             "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/rules_pkg-0.2.0.tar.gz",
             "https://github.com/bazelbuild/rules_pkg/releases/download/0.2.0/rules_pkg-0.2.0.tar.gz",
         ],
+        # bazelbuild/rules_proto
+        "b0cc14be5da05168b01db282fe93bdf17aa2b9f4.tar.gz": [
+            "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/b0cc14be5da05168b01db282fe93bdf17aa2b9f4.tar.gz",
```

Comment:
This will also need to be uploaded. Thanks in advance!

Reply:
Can you create a script like https://github.com/bazelbuild/bazel/blob/master/tools/android/runtime_deps/upload_android_tools.sh to automate updating this?

Reply:
Created `tools/upload_bazel_rules.sh`